### DISCUSSION TITLE
Feature Request: Provide options to filter image/video/text only

### DISCUSSION THREAD

**Discussion by wisecoders** [robertodoering/harpy#837] [Ideas]
It will be really amazing if we are able to filter the type of tweets we would like to see. Please give options to filter type of tweets

**Comment by robertodoering** [comment]
Can you elaborate a bit what set of filters you'd like to see that aren't yet possible?

**Comment by wisecoders** [comment]
Just now saw the filter option and pls close this one. Sorry for it.

**Comment by wisecoders** [comment]
Just now saw the filter option and close this one. Sorry for it.
